bin/asset-pipeline

```diff
@@ -162,7 +162,7 @@ class AssetPipeline {
 
   async rewriteJS(origPath, key, assetMap = this.assetMap) {
     let code = await fs.readFile(origPath, 'utf8');
-    const ast = acorn.parse(code, { ecmaVersion: 6 });
+    const ast = acorn.parse(code, { ecmaVersion: 8 }); // 2017
```

Comment:
Is this ok? I'd like to use `async` functions.. (caniuse says it's fine)

Reply:
Yeah that seems reasonable to me.